website/content/docs/enterprise/ent-to-ce-downgrades.mdx

```diff
@@ -0,0 +1,144 @@
+---
+layout: docs
+page_title: Downgrade from Consul Enterprise to the community edition
+description: >-
+  Learn how to downgrade your installation of Consul Enterprise to the free Consul community edition (CE)
+---
+
+# Downgrade from Consul Enterprise to the community edition
+
+This document describes how to downgrade from Consul Enterprise to Consul community edition (CE).
+
+## Download the CE binary version
+
+First, download the binary for the CE version you want.
+
+<Tabs>
+<Tab heading="Binary">
+
+All current and past versions of the CE and Enterprise releases are
```

Comment:
It would be good to instead provide a curl command that downloads a version of Consul for example. 


```
export VERSION=1.17.0
curl https://releases.hashicorp.com/consul/1.17.0/consul_${VERSION}_linux_amd64.zip
```